src/setup.lambda.ts

```diff
@@ -55,11 +55,14 @@ async function handleDomain(event: ApiGatewayEvent): Promise<AWSLambda.APIGatewa
   if (!body.domain) {
     return response(400, 'Invalid domain');
   }
+  if (!body.runnerLevel) {
```

Comment:
We should have the default still be `repo` for backwards compatibility.

Reply:
Fine